contracts/multisig/src/events.rs

```diff
@@ -15,6 +16,7 @@ pub enum Event {
         worker_set_id: String,
         pub_keys: HashMap<String, PublicKey>,
         msg: MsgToSign,
+        chain_name: ChainName,
```

Comment:
Why?

Reply:
axelarons needs to know which chain a session was for. This is not necessarily the event they are going to listen to, but I think it's a good addition

Reply:
also ampd needs to know, because singing needs to be done differently for certain chains